### PR TITLE
ES6 Arrow functions sample (fixes #148)

### LINKED ISSUE #148: Arrow functions (ES6) sample
See https://www.chromestatus.com/feature/5047308127305728

### PR DESCRIPTION
**Pull request by addyosmani**
r: @jeffposnick

### CODE REVIEW

File: arrows-es6/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+additional_head_content: <meta name="optional" value="optional">
+---
+
+<h3>Background</h3>
+
+<p>ES6 fat arrow functions have a shorter syntax compared to
+function expressions and lexically bind the <code>this</code> value. Arrow functions are always anonymous and effectively turn
+<code>function (arguments) { expression }</code> into arguments => expression. If using an expression after an arrow, the return is implicit, so no <code>return</code> is required.</p>
+
+{% capture initial_output_content %}
```
Comment (jeffposnick) [inline arrows-es6/index.html]: You can remove this `{% capture initial_output_content %}` block. (It's just needed if you want something initial to show up in the output area.)

File: arrows-es6/index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+chrome_version: 45
+feature_id: 5047308127305728
+additional_head_content: <meta name="optional" value="optional">
+---
+
+<h3>Background</h3>
+
+<p>ES6 fat arrow functions have a shorter syntax compared to
+function expressions and lexically bind the <code>this</code> value. Arrow functions are always anonymous and effectively turn
+<code>function (arguments) { expression }</code> into arguments => expression. If using an expression after an arrow, the return is implicit, so no <code>return</code> is required.</p>
```
Comment (jeffposnick) [inline arrows-es6/index.html]: `<code>arguments => expression</code>`

File: arrows-es6/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  setInterval(() => this.seconds++, 1000);
+}
+
+let counterB = new counter_es6();
+setTimeout(() => ChromeSamples.log(counterB.seconds), 1200);
+
+{% endcapture %}
+{% include js_snippet.html js=js %}
+
+{% include css_snippet.html css=css %}
```
Comment (jeffposnick) [inline arrows-es6/index.html]: You can leave out `{% include css_snippet.html css=css %}`. It's just needed if there is inline CSS that you're demonstrating as part of the sample.

File: arrows-es6/index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
+---
+feature_name: ES6 Arrow functions
+chrome_version: 45
+feature_id: 5047308127305728
+additional_head_content: <meta name="optional" value="optional">
```
Comment (jeffposnick) [inline arrows-es6/index.html]: This isn't needed—it's just showing off how to include extra tags in the `<head>`.

**Comment by jeffposnick** [comment]
Thanks, reviewed. Sorry that the starter template leaves some confusion about what's required vs. what's just there to be illustrative.

**Comment by addyosmani** [comment]
No worries at all. We launch and iterate :) Feedback addressed. PTAL when you have a chance.

**Comment by jeffposnick** [comment]
:+1:

**Pull request merged**